app/controllers/concerns/hyrax/collections_controller_behavior.rb

```diff
@@ -103,6 +103,8 @@ def create
       @collection.attributes = collection_params.except(:members)
       @collection.apply_depositor_metadata(current_user.user_key)
       add_members_to_collection unless batch.empty?
+      @collection.visibility = Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PRIVATE
```

Comment:
I wonder if for security purposes, the same check should happen for #update, even though it should already be set there.